src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java

```diff
@@ -374,4 +374,13 @@ static boolean matchesFilter(Block block, ItemStack item, int index) {
         }
     }
 
+    /**
+     * Get the whitelist/blacklist slots in a Cargo Input Node. If you wish to access the items
+     * in the cargo (without hardcoding the slots in case of change) then you can use this method.
+     *
+     * @return The slot indexes for the whutelist/blacklist section.
+     */
+    public static int[] getSlots() {
```

Comment:
getSlots() seem rather un-intuitive. Think that name could be improved.

Reply:
getWhitelistBlacklistSlots? It's always awkward to name

Reply:
I know, but getSlots() is rather misleading. getCargoConfigArea() maybe?